### DISCUSSION TITLE
Interpolate $__interval and $__interval_ms in data links

### DISCUSSION THREAD

**Discussion by glep207** [grafana/grafana#42378] [Dashboards, Panels & Visualizations]
Suppose I have a time series of aggregated data. I want to define a data link such that when I click on a data point, I navigate to another dashboard where I can see all measurements that had been aggregated to a single data point in the first dashboard.

To do this the timeframe of the data link should be `time=${__value.time}&time.window=${__interval_ms}`. 

The problem is that as of Grafana 8.1 neither ${__interval_ms} or $__interval are interpolated in the data link.

See also https://community.grafana.com/t/link-to-external-system-using-current-interval/721, and somewhat related issues #15168, #38252 and #39249

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
